cypress-unit.json

```diff
@@ -1,11 +0,0 @@
-{
-  "$schema": "https://on.cypress.io/cypress.schema.json",
```

Comment:
Were there no component tests to run?

Reply:
looks like there were a few:
- https://github.com/cypress-io/cypress-example-todomvc-redux/blob/master/src/reducers/todos.spec.js 
- https://github.com/cypress-io/cypress-example-todomvc-redux/tree/master/src/components
seem like we need to migrate to use ct-testing? or do we need a follow-up issue to implement in a diff pr?

Reply:
I'm prioritizing updating to cypress 12 over converting old component tests to new style. I briefly looked into it and quit when it wanted me to setup webpack (which the project doesn't use other than to satisfy the preprocessor peer dependency).

Reply:
👍🏻 maybe just log an issue to set up Component testing and we can get it routed to CT for better coverage / examples

Reply:
https://github.com/cypress-io/cypress-example-todomvc-redux/issues/243